Digitisers/SimHitMerge/src/SimHitMergeAlg.cpp

```diff
@@ -56,17 +57,20 @@ StatusCode SimHitMergeAlg::initialize() {
 
 StatusCode SimHitMergeAlg::execute()
 {
-
+     bool sanity_check = true;
```

Comment:
It looks like this variable is always true. Do you plan to change it to a property?

Reply:
Yes, In future it will be a property.